package.json

```diff
@@ -62,12 +63,13 @@
     "node-fetch": "^2.6.0",
     "openfin-service-async": "^1.0.1",
     "openfin-service-signal": "^1.0.0",
-    "openfin-service-tooling": "1.0.22",
+    "openfin-service-tooling": "^1.0.24",
```

Comment:
Can you remove the `^` here? Service tooling isn't semver, so should be locked to an exact version.